frontend/src/app/components/medical-record-wizard/medical-record-wizard.component.scss

```diff
@@ -0,0 +1,5 @@
+.medical-record-wizard {
```

Comment:
view encapsulation shouldn't be necessary if you just remove the `.medical-record-wizard` wrapping class and call it something unique